terraform-provider-constellation/internal/provider/provider_test.go

```diff
@@ -34,6 +34,10 @@ var testAccProtoV6ProviderFactories = map[string]func() (tfprotov6.ProviderServe
 // bazelSetTerraformBinaryPath sets the path to the Terraform binary for
 // acceptance testing when running under Bazel.
 func bazelSetTerraformBinaryPath(t *testing.T) {
+	if !runsUnderBazel() {
+		return
+	}
+
```

Comment:
```suggestion
```

We do this check below already:
https://github.com/edgelesssys/constellation/blob/97aea98e7771d916a234707b03ef6b8df9427e3f/terraform-provider-constellation/internal/provider/provider_test.go#L41-L50